### DISCUSSION TITLE
For the "cards stack" example, when cards are flicked to the right, they are still visible if you scroll the window, are there any fixes?

### DISCUSSION THREAD

**Discussion by demonhunter3333** [pmndrs/react-spring#1704] [Support] [answered]
First off, I'd like to express my appreciation for the great work that went into developing this amazing library!

For the "cards stack" example, when cards are flicked to the right, they are still visible if you scroll the window, are there any fixes?

https://codesandbox.io/s/github/pmndrs/react-spring/tree/master/demo/src/sandboxes/cards-stack

<img width="551" alt="Screenshot 2021-10-04 at 8 42 34 PM" src="https://user-images.githubusercontent.com/43437154/135853287-f9f7aa27-eb78-4609-a63b-4d691ac56f01.png">

**Comment by joshuaellis** [comment, marked as answer]
Yes. If you use CSS and put:
```css
position: relative;
overflow: hidden;
```
on the container class, this will stop.

See this – https://codesandbox.io/s/musing-khayyam-z0kyg?file=/src/styles.module.css

**Reply by demonhunter3333** [reply]
It worked! thank you!
